<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actopayreport</class>
 <widget class="QDialog" name="actopayreport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonBuild">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>641</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string> Filtrar Por</string>
   </property>
   <widget class="QGroupBox" name="groupBoxFilterData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>310</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string> Data </string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title">
      <string> Tipo </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonFilterIssue">
        <property name="text">
         <string>Lançamento</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFilterVencto">
        <property name="text">
         <string>Vencimento</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFilterPayment">
        <property name="text">
         <string>Pagamento</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>25</y>
       <width>171</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>De</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEditFrom">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Até</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEditUntil">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterApenasDe">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>306</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string> Apenas De </string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fornecedor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxFornecedor">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterSituation">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>180</y>
      <width>290</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string> Situação </string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxPaid">
       <property name="text">
        <string>Pagas</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxNotPaid">
       <property name="text">
        <string>Em Aberto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxTipoDeConta">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Tipo de Conta</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBoxTipodeConta">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>310</width>
      <height>96</height>
     </rect>
    </property>
    <property name="title">
     <string>Ordenar Por</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxOrderIssue">
       <property name="text">
        <string>Lançamento</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxOrderPessoa">
       <property name="text">
        <string>Fornecedor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxOrderVncto">
       <property name="text">
        <string>Vencimento</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxOrderSituation">
       <property name="text">
        <string>Situação</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxOrderPayment">
       <property name="text">
        <string>Pagamento</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxOrderTipo">
       <property name="text">
        <string>Tipo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
